src/pages/Timeline/CreatePost.jsx

```diff
@@ -85,7 +85,7 @@ export default function CreatePost({ loadTimelinePosts }) {
 			<img src={userInfo.userImg} alt='perfil' />
 			<PostContent>
 				<form onSubmit={publishPost}>
-					<h2>O que você tem pra favoritar hoje?</h2>
+					<h2>What do you have to favorite today?</h2>
```

Comment:
Acho que a gente poderia mudar isso pra algo do tipo
What do you want to share today?
What you'd like to share today?
What would you like to share today?

Reply:
Boa! Gostei do 'What you'd like to share today?'